src/coreclr/vm/stackwalk.cpp

```diff
@@ -553,6 +553,13 @@ PCODE Thread::VirtualUnwindCallFrame(T_CONTEXT* pContext,
     UINT_PTR            uImageBase;
     PT_RUNTIME_FUNCTION pFunctionEntry;
 
+#if !defined(TARGET_UNIX) && defined(CONTEXT_UNWOUND_TO_CALL)
```

Comment:
When is `CONTEXT_UNWOUND_TO_CALL` not defined on Windows here?

I see `CONTEXT_UNWOUND_TO_CALL` defined for both x64 and arm64 in Windows SDK.

Reply:
Hmm, that's strange. In pal.h we don't define it for x64 and Windows also doesn't set it for non-arm.

Reply:
I've added a commit to change that ifdef to TARGET_ARM64